content/commands/hexpire/index.md

```diff
@@ -71,7 +71,7 @@ key_specs:
     type: range
   update: true
 linkTitle: HEXPIRE
-since: 8.0.0
+since: 7.4 RC1
```

Comment:
Please change to 7.4 or 7.4.0 (for all commands).

Reply:
Done. Next push.